.pre-commit-config.yaml

```diff
@@ -43,3 +43,16 @@ repos:
     rev: 23.1.0
     hooks:
       - id: black
+  - repo: https://github.com/asottile/add-trailing-comma
+    rev: v3.0.0
+    hooks:
+      - id: add-trailing-comma
+  - repo: https://github.com/MarcoGorelli/absolufy-imports
```

Comment:
While I agree this is a helpful idea, I'm a bit nervous about an automated tool that updates the imports for me rather than warnings to change something. Second this doesn't seemed to have changed any code from what I can see.
Therefore, could you remove